TemplePlus/action_sequence.cpp

```diff
@@ -229,10 +232,6 @@ ActionSequenceSystem::ActionSequenceSystem()
 	//rebase(GetRemainingMaxMoveLength, 0x1008B8A0);
 	//rebase(TrimPathToRemainingMoveLength, 0x1008B9A0);
 
-	rebase(_CrossBowSthgReload_1008E8A0, 0x1008E8A0);
-
-	rebase(ActionCostReload, 0x100903B0);
```

Comment:
I thought we already have `stdAttackAddToSeq` reimplemented...

![image](https://github.com/user-attachments/assets/97748ae0-0158-4ce1-a1ee-9df1484b9c89)

Reply:
We do. I'm not sure what the question is.

I replaceFunctioned `ActionCostReload` in case it's called in some DLL spot we haven't replaced. That isn't the only one, if I recall.

Reply:
This is where it's used:

![image](https://github.com/user-attachments/assets/98eb03e2-c792-4299-89e1-8b220c4394af)

Reply:
Best assigned here for D20A_RELOAD, like this:

![image](https://github.com/user-attachments/assets/fba1eee7-d30c-44a5-a3a4-e95f3277f9f2)

Reply:
Therefor this function substitute hack could be removed altogether.

Reply:
I guess now all the occurrences from the original DLL have been replaced, because I added the reload action to the new `d20Defs`. So yeah, the `replaceFunction` is unnecessary.